Documentation/Contributors/PerformanceTestingGuide/README.md

```diff
@@ -0,0 +1,235 @@
+  url,
+});
+tileset.maximumScreenSpaceError = 4;
+// and any other initialization code
+
+viewer.scene.primitives.add(tileset);
+
+tileset.readyPromise.then(() => {
+  tilesetTimer.stop();
+  tilesetTimer.print();
+});
+```
+
+### How to measure tile load time
+
+To time how long it takes for all the tiles in the current camera view to load
+(not including the tileset load time), start the timer in the ready promise
+and stop it in the `initialTilesLoaded` event handler. This event is used
+because we only care about our initial fixed camera view. `allTilesLoaded`, in
+contrast, may trigger multiple times if the camera moves, which is undesireable
+here.
+
+```js
+tileset.readyPromise.then(() => {
+  tileTimer.start();
+});
+
+// This callback is fired the first time that all the visible tiles are loaded.
+// It indicates the end of the test.
+tileset.initialTilesLoaded.addEventListener(() => {
+  tileTimer.stop();
+  tileTimer.print();
+});
+```
+
+### Caching vs throttling
+
+Depending on the scenario you are trying to test for, you may want to enable
+caching of network requests or throttling network speed.
+
+- If you want to simulate slow network connections, consider throttling the
+  connection. For example, in Chrome, see [these instructions](https://developer.chrome.com/docs/devtools/network/#throttle)
+- If you want to eliminate network latency and focus on measuring how long the
+  initialization code takes to run, enable caching. In Chrome, this option
+  can be done by _unchecking_ the `Disable Cache` option in the Network tab.
+  You will also need to enable caching on the server.
+
+![Uncheck disable cache](no-disable-cache.png)
+
+To check if caching is correctly enabled, load the tileset and check the
+Network tab. In Chrome, cached files will show up as `(disk cache)` in the
+`Size` column:
+
+![Disk Cache](disk-cache.png)
+
+### Loading tilesets with a static server
+
+If using tilesets locally from disk, you will need a static server to load
+tilesets. Most any static server could work, though be sure to consider the
+following:
+
+1. If caching is used, this must be enabled on the server. Note that the
+   CesiumJS development server (`npm run start`) does not support caching.
+2. If the static server is listening on a different domain or port, make sure
+   to enable Cross-Origin Resource Sharing (CORS)
+
+A example using the `npm` module `http-server` might look like this:
+
+```bash
+# Host the current directory at http://localhost:8003
+# http-server defaults to caching with max-age of 1 hour.
+# to disable caching, add the -c-1 flag
+http-server -p 8003 --cors
+```
+
+### Running the Test
+
+When running the test, there are a few additional considerations:
+
+1. Use the built version of Sandcastle to remove debug pragmas, combine JS
+   files, and minify the results. This way, results reflect how users will
+   be running Cesium.
+
+```
+# first time only: make a release build of CesiumJS
```

Comment:
This may be a bit misleading. You'll need to run release again when switching branches or after modifying code.